README.md

```diff
@@ -0,0 +1,66 @@
+# go-da
+
+go-da defines a generic Data Availability interface for modular blockchains.
+
+Note that the rollup clients _do not_ need to depend on any implementation,
```

Comment:
this paragraph is not flowing very well. i would get rid of it completely and directly jump to the interface methods.